packages/amplify-e2e-core/src/categories/geo.ts

```diff
@@ -44,23 +44,17 @@ export function addMapWithDefault(cwd: string, settings: GeoConfig = {}): Promis
       .wait('Who can access this Map?')
       .sendCarriageReturn();
 
-    chain.wait('Do you want to configure advanced settings?').sendConfirmNo();
+    chain.wait('Do you want to configure advanced settings?').sendNo();
 
     if (config.isAdditional === true) {
       chain.wait(defaultMapQuestion);
       if (config.isDefault === true) {
-        chain.sendConfirmYes();
+        chain.sendYes();
       } else {
-        chain.sendConfirmNo();
+        chain.sendNo();
       }
     }
-    chain.run((err: Error) => {
-      if (!err) {
-        resolve();
-      } else {
-        reject();
-      }
-    });
+    chain.runAsync();
```

Comment:
need to remove the `return new Promise(...` wrapper and return `chain.runAsync()` (applies to other `runAsync` usages as well)